### PR TITLE
CI: Include 2.6, 2.7, as rvm aliases, drop <2.5

### PR DESCRIPTION
**Pull request by olleolleolle**
This matrix now reflects the supported Ruby versions, see https://www.ruby-lang.org/en/downloads/branches/ for version policy of Ruby.

Using the rvm aliases keeps the matrix up-to-date with the latest patches in a minor series.

Hope this helps!

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
 
 rvm:
-  - 2.4.5
-  - 2.5.0
-  - 2.5.1
-  - 2.5.2
-  - 2.5.3
+  - "2.5"
+  - "2.6"
+  - "2.7"
```
Comment (elalemanyo) [inline .travis.yml]:
```suggestion
  - 2.5
  - 2.6
  - 2.7
```
Looking to travis documentation I like more this way to write it.
https://docs.travis-ci.com/user/build-matrix/#matrix-expansion

What do you think?

Comment (olleolleolle) [reply .travis.yml]: I'll make the change; this detail is mostly "avoid Float" (for non-Travis YAML tools to be able to consume the file).

Comment (elalemanyo) [reply .travis.yml]: non-Travis? but this is settings file for travis, right?

Comment (olleolleolle) [reply .travis.yml]: Change applied! ✨

**Comment by elalemanyo** [comment]
@olleolleolle thanks for updating this!

**Comment by olleolleolle** [comment]
Shoutout the RUG::B for noting this gem!

**Pull request merged**